base/challenge-skeleton/challenge/config/nsjail.cfg

```diff
@@ -38,16 +38,6 @@ mount: [
     src: "/config"
     dst: "/config"
     is_bind: true
-  },
```

Comment:
we dont need the other settings?

Reply:
infra-wide i mean

Reply:
which other settings do you mean?

Reply:
uidmap, chroot

Reply:
ah, that doesn't work. See the thread with robert and wiktor